pkg/eks/components/CruEKS.vue

```diff
@@ -596,7 +635,7 @@ export default defineComponent({
         <LabeledInput
           required
           label-key="eks.clusterName.label"
-          :value="config.displayName"
+          :value="normanCluster.name"
```

Comment:
config.displayName = eks cluster name in aws
normanCluster.name = rancher cluster object's name